api/src/models/User.ts

```diff
@@ -29,6 +39,50 @@ module.exports = (sequelize: Sequelize) => {
         defaultValue: DataTypes.UUIDV4,
         primaryKey: true,
       },
+      
+      UUID: {
+        type: DataTypes.UUID,
+        defaultValue: DataTypes.UUIDV4,
+      }, 
+
+      firstName: {
+        type: DataTypes.STRING,
+        defaultValue: DataTypes.STRING,
+      },
+
+      lastName: {
+        type: DataTypes.STRING,
+        defaultValue: DataTypes.STRING,
+      },
+      
+      username: {
+        type: DataTypes.STRING,
+        defaultValue: DataTypes.STRING,
+      },
+
+      email: {
+        type: DataTypes.STRING,
+        defaultValue: DataTypes.STRING,
+      },
```

Comment:
De preferencia añadir una propiedad `validate` que confirme que es un correo (consultar la documentacion de Sequelize para mas detalles.)
```suggestion
      email: {
        type: DataTypes.STRING,
        allowNull: false,
        validate: {
          isEmail: true,
        },
      }
```